web-common/src/main/java/edu/unc/lib/boxc/web/common/services/SolrQueryLayerService.java

```diff
@@ -182,6 +182,7 @@ public Map<String, Long> getFormatCounts(AccessGroupSet accessGroups) {
             query.setRows(0);
             query.addFacetField(SearchFieldKey.FILE_FORMAT_CATEGORY.getSolrField());
             query.setFacetLimit(-1);
+            query.addFilterQuery("resourceType:(Work)");
```

Comment:
It would be best to use the constants when possible, in case fields or types get renamed later:
`SearchFieldKey.RESOURCE_TYPE.getSolrField()`
and
`ResourceType.Work.name()`

Reply:
so would it be something like `query.addFilterQuery(SearchFieldKey.RESOURCE_TYPE.getSolrField() + ":(" + ResourceType.Work.name() + ")");`?

Reply:
yes, good old java string concatenation